src/main/java/mousio/etcd4j/transport/EtcdNettyClient.java

```diff
@@ -392,7 +413,6 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws E
   @Override
   public void close() {
     logger.info("Shutting down Etcd4j Netty client");
-    this.timer.stop();
```

Comment:
Daemon is ok but I would like to be explicit in resource cleanup and I'd like to keep the same semantic as per eventloop so one can safely delegate timer's resource cleanup to etcd4j no matter of how the thread are configured.